### ISSUE TITLE
Huoltotunneli most likely allows for car traffic where it should not

### ISSUE THREAD

**Issue by hsl-petrhaj** [HSLdevcom/helmet-model-system#533]
Huoltotunneli is mostly for logistics except for the western part. Nowadays cars can technically go through the tunnel in the model, although the speed is reduced to 15 kph. If the surface streets are heavily congested it can still be the fastest option. 
Some checks need to be done and changes applied either to "Helmet 4.2" or straight to Helmet 5. Since it might break backward compatibility, Helmet 5 should be the way IMHO.

Huoltotunneli and its encoding is described here:
[Emme network](https://hsldevcom.github.io/helmet-docs/emme_verkko.html)
The information is partially outdated, because at least "h" mode should be added.
I made issue about this here: [Helmet docs issue](https://github.com/HSLdevcom/helmet-docs/issues/14)

So far, there only seems to be scenarios with 0 cars as a result (2018,2040), but could some "horror scenario" cause some volumes to go through there? I think it needs some testing at least.

**Comment by hsl-petrhaj** [comment]
Adding a picture of test with some less probable scenario (no traffic on Espa and Kaivokatu)

![kuva](https://github.com/HSLdevcom/helmet-model-system/assets/127391244/c068ea36-eeda-420f-bc7f-0d7eba2b151f)

Note: This was run only via Desktop's SOLA traffic assignment. Some issues with Emme Python at the moment.

**Comment by hsl-petrhaj** [comment]
@johpiip 
@zptro

**Comment by johpiip** [comment]
I have no input on this issue. 😕

**Comment by zptro** [comment]
I do not think this an issue. People do use the tunnel when streets are very congested, and now there has even been discussion about allowing this (if the user pays a fee).

**Comment by hsl-petrhaj** [comment]
"c" mode is not allowed through in Helmet 5 networks, thus problem is solved.
